### DISCUSSION TITLE
Removal of someone else's repo.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#72090] [Repositories] [answered]
### Select Topic Area

Question

### Body

I have made a pull request in the hacktoberfest, but that repository is showing on my repositories tab and there is no option to delete it.
Please help
![image](https://github.com/community/community/assets/113282831/1595827b-a194-4914-9e43-1c3686032e90)

see the last repo, its not mine

**Comment by Alfedi** [comment, marked as answer]
It's because you are now a contributor to the repo. I believe there is no way to remove it from the sidebar

**Reply by ghost** [reply]
Then is there a way to remove myself as a collaborator?
I have mailed the owner though.

**Reply by Alfedi** [reply]
The owner can, but I don't see the point doing it. 🤷
You will lose the credit for having contributed something to the repository. If you don't care about that go on then.

**Reply by ghost** [reply]
Ok, thanks :)

**Comment by queenofcorgis** [comment]
We’re glad you could get this solved and we want to make sure others can do the same so we’ve reopened this discussion for better discoverability for our members.
